src/connector.ts

```diff
@@ -727,7 +731,6 @@ export async function connect(
         }
     } catch (e) {
         console.log(e);
```

Comment:
@PEZ It might be a good idea to show a window error message here to the user indicating that an error occurred while connecting. We might also want to use `console.error` instead of `console.log` here. Just adding this comment here for later.

Reply:
I don't know in which situations this actually fails. I thought it better to not alarm the user and ask for the port instead. Agree about `console.error`!